MvvmCross/Platforms/WinUi/Presenters/Models/WindowInformation.cs

```diff
@@ -0,0 +1,86 @@
+#nullable enable
+using MvvmCross.ViewModels;
+using Microsoft.UI.Xaml;
+using System.Collections.Concurrent;
+using MvvmCross.Platforms.WinUi.Views;
+
+namespace MvvmCross.Platforms.WinUi.Presenters.Models
+{
+    /// <summary>
+    /// Holds information regarding the different windows.
+    /// </summary>
+    public class WindowInformation
```

Comment:
Seal this if no inheritance is intended

Reply:
Done. I also made it internal as it shouldn't be used outside the library.